common/source/docs/common-polygon_fence.rst

```diff
@@ -1,64 +1,133 @@
 .. _common-polygon_fence:
 
-[copywiki destination="copter,rover"]
+
 ==============================
 Inclusion and Exclusion Fences
 ==============================
 
 Overview
 ========
 
-Copter and Rover includes support for polygon fences with up to 70 points and circular fences.  Either may be selected to be inclusion or exclusion type. The purpose of these fences are to attempt to stop your vehicle from flying into (exclusion), or out of (inclusion), the fences by initiating a failsafe action like RTL or, if flying in Loiter mode, the vehicle will normally stop before breaching the fence
+ArduPilot includes support for polygon fences with up to 70 points and for pure circular fences with specified radii. Unlike the home-based cylinderical fence :ref:`cylindrical fence <common-ac2_simple_geofence>`, the circular inclusion/exclusion fence can be placed anywhere. Either the polygonal or circular fences may be selected to be inclusion or exclusion type, and may be mixed. The purpose of these fences are to attempt to stop your vehicle from flying into (exclusion), or out of (inclusion), the fences by initiating a failsafe action like RTL or, if flying in Loiter mode for Copter, the vehicle will normally stop before breaching the fence.
```

Comment:
Phrasing here is a little weird, especially at `...or out of (inclusion), the fences...`

`The purpose of these fences are to attempt to stop your vehicle from flying into (exclusion), or out of (inclusion), the fences by initiating a failsafe action like RTL or, if flying in Loiter mode for Copter, the vehicle will normally stop before breaching the fence.`

Reply:
Hmm, after a coffee this makes a little more sense.

FYI, by default Copter will stop at the fence, but if AVOID_ENABLE has fence avoidance turned off, then you can in fact breach the fence.

Reply:
expanded